Sources/StreamChatUI/MessageActionsPopup/MessageActionsTransitionController.swift

```diff
@@ -12,20 +12,51 @@ open class MessageActionsTransitionController: NSObject, UIViewControllerTransit
     /// Indicates if the transition is for presenting or dismissing.
     open var isPresenting: Bool = false
     /// `messageContentView`'s initial frame.
+    @available(
+        *,
+        deprecated,
+        message: "This field is no longer used because the message content view is not moved across view heirarchies anymore but get copied."
+    )
     open var messageContentViewFrame: CGRect = .zero
     /// `messageContentView`'s constraints to be activated after dismissal.
+    @available(
+        *,
+        deprecated,
+        message: "This field is no longer used because the message content view is not moved across view heirarchies anymore but get copied."
+    )
     open var messageContentViewActivateConstraints: [NSLayoutConstraint] = []
     /// Constraints to be deactivated after dismissal.
+    @available(
+        *,
+        deprecated,
+        message: "This field is no longer used because the message content view is not moved across view heirarchies anymore but get copied."
+    )
     open var messageContentViewDeactivateConstraints: [NSLayoutConstraint] = []
     /// `messageContentView` instance that is animated.
     open weak var messageContentView: ChatMessageContentView?
     /// `messageContentView`'s initial superview.
+    @available(
+        *,
+        deprecated,
+        message: "This field is no longer used because the message content view is not moved across view heirarchies anymore but get copied."
+    )
     open weak var messageContentViewSuperview: UIView?
     /// Top anchor for main container.
+    @available(
+        *,
+        deprecated,
+        message: "This field is no longer used because the message content view is not moved across view heirarchies anymore but get copied."
+    )
```

Comment:
I don't think it is valid that we these properties as deprecated, because they are actually not working anymore. When something is deprecated, it means it still works, but it won't work in the future

Reply:
that's actually a very good point. @evsaev what do you say:
1. we implement a new `MessageActionsTransitionController`
2. if the user assigned their subclass to `Components`, we use it, otherwise we use the new, improved version

Reply:
@b-onc the `MessageActionsTransitionController` is not part of `Components` yet and the reason for this is the transition being quite complex. I assume none of the customers has this part customized so removing those fields is actually OK. Not that high price for the 2 issues we've addressed.

Reply:
✅ Added transition controller to `Components`